--- a/email.service.js
+++ b/email.service.js
@@ -69,6 +69,8 @@
         const shareUrl = `${this.config.get("general.appUrl")}/s/${shareId}?recipient=${encodeURIComponent(recipientId)}`;
         const lang = this.config.get("general.defaultLanguage");
         const locale = this.i18n.translate("email.locale", { lang });
+        const trimmedDesc = (description ?? "").trim().replace(/\.+$/, "");
+        const descBlock = trimmedDesc ? ` en dit bericht werd toegevoegd: "${trimmedDesc}".` : ".";
         let replyTo = undefined;
         if (this.config.get("email.shareRecipientsReplyToCreator") &&
             creator?.email) {
@@ -86,10 +88,11 @@
             creatorEmail: creator?.email ?? "",
             shareUrl,
             desc: description ?? this.i18n.t("email.shareRecipientsDescFallback"),
+            descBlock,
             expires: moment(expiration).unix() != 0
                 ? moment(expiration).locale(locale).fromNow()
-                : this.i18n.t("email.shareRecipientsExpiresNeverFallback"),
+                : "nooit",
             name: recipientName ?? "",
             recipient: recipientName ?? "",
             recipientEmail,
             email: recipientEmail,
